configs/solc-bin-ci-reference.json

```diff
@@ -0,0 +1,16 @@
+{
+  "binaries": {
+    "VERSION": {
+      "is_enabled": true,
+      "protocol": "https",
+      "source": "https://github.com/matter-labs/era-solidity/releases/download/VERSION/solc-${PLATFORM}-VERSION",
```

Comment:
Could you use the similar syntax for `VERSION`: `${VERSION}`?

Reply:
I was planning to use `sed` to change `VERSION` string, it's not for a variable expansion. So, I can't use variable expansion for `${VERSION}` here.

Reply:
I don't think there is a better solution with env variables somehow, so I have to `sed` in CI.

Reply:
The other one is also not for variable expansion, as it is replaced by compiler-tester, but having two kinds of undocumented variables in one small file is already too much.